.github/workflows/release_macOS.yml

```diff
@@ -86,3 +95,64 @@ jobs:
         run: yarn release:mac:arm64
         env:
           NOTARIZE: true
+      - name: Copy dist/latest-mac.yml to dist/latest-mac-arm64.yml
+        run: |
+          cp dist/latest-mac.yml dist/latest-mac-arm64.yml
+        shell: bash
+      - name: Save latest-mac-arm64.yml as an artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: latest-mac-arm64
+          path: dist/latest-mac-arm64.yml
+
+  generate-latest-mac:
+    needs:
+      - release-x64
+      - release-arm64
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+          cache: 'yarn'
+
+      - name: Reconfigure git to use HTTP authentication
+        run: >
+          git config --global url."https://github.com/".insteadOf
+          ssh://git@github.com/
+
+      - name: Authenticate with private NPM package
+        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
+
+      - name: Install dependencies
+        run: yarn setup
+
+      - name: Download artifacts from release-x64 and release-arm64 jobs
+        uses: actions/download-artifact@v2
+        with:
+          name: latest-mac-x64
+          path: dist-x64
+
+      - uses: actions/download-artifact@v2
+        with:
+          name: latest-mac-arm64
+          path: dist-arm64
+
+      - name: Merge YMLs
+        run: |
+          npm install js-yaml
```

Comment:
curious why we did npm instead of yarn here

Reply:
copy paste from chatgpt, changed just a few things and left this one here.
I will do some cleanup and update the manifests later